packages/widget/src/stores/settings/useSettingsStore.ts

```diff
@@ -142,6 +142,16 @@ export const useSettingsStore = createWithEqualityFn<SettingsState>(
         get().initializeTools('Bridges', bridges, true);
         get().initializeTools('Exchanges', exchanges, true);
       },
+      getSettings: () => ({
+        appearance: get().appearance,
+        gasPrice: get().gasPrice,
+        language: get().language,
+        routePriority: get().routePriority,
+        enabledAutoRefuel: get().enabledAutoRefuel,
+        slippage: get().slippage,
+        enabledBridges: { ...get()._enabledBridges },
+        enabledExchanges: { ...get()._enabledExchanges },
```

Comment:
I've went with using the '_enabledBridges' and '_enabledExchanges' properties as being object maps they contain all the info that the arrays present anyway - they present the list of chains and whether they are enabled or disabled.

Also using the object rather than an array produces much simpler diffing information. With using the object for example, when changing a single bridge from enabled to disabled we only get one entry in the diff - using an array we would get that change entry in the diff and in additional a change of all the indexes of the other items in the array.